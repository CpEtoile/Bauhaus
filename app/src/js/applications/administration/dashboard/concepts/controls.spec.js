import React from 'react';
import { render } from '@testing-library/react';
import ConceptDashboardControls from './controls';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

describe('concept-visualization-controls', () => {
	it('renders without crashing', () => {
		render(
			<MemoryRouter>
				<ConceptDashboardControls />
			</MemoryRouter>
		);
	});
	it('should render the component', () => {
		const tree = renderer
			.create(
				<MemoryRouter>
					<ConceptDashboardControls />
				</MemoryRouter>
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
