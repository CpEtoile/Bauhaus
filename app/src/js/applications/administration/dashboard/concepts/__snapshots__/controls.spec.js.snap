// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`concept-visualization-controls should render the component 1`] = `
<div
  className="row btn-line action-toolbar"
>
  <div
    className="col-md-2"
  >
    <a
      className="btn btn-primary btn-lg col-md-12"
      href="/concepts/administration"
      onClick={[Function]}
    >
      Back
    </a>
  </div>
</div>
`;
